src/Compilers/VisualBasic/Test/Emit/Resource.Designer.vb

```diff
@@ -1,255 +1,255 @@
+        '''        Dim UI As UInteger
+        '''        Dim Lo As Long
+        '''        Dim UL As ULong
+        '''        Dim De As Decimal
+        '''        Dim Si As Single
+        '''        Dim [Do] As Double
+        '''        Dim SBZero As S [rest of string was truncated]&quot;;.
+        '''</summary>
+        Friend Shared ReadOnly Property ConversionsILGenTestSource1() As String
+            Get
+                Return ResourceManager.GetString("ConversionsILGenTestSource1", resourceCulture)
+            End Get
+        End Property
+        
+        '''<summary>
+        '''  Looks up a localized string similar to Option Strict Off
+        '''Imports System
+        '''Imports System.Collections.Generic
+        '''Imports System.Globalization
+        '''
+        '''Module Module1
+        '''
+        '''
+        '''    Sub Main()
+        '''        System.Console.WriteLine(&quot;Conversions from Nothing literal:&quot;)
+        '''
+        '''        PrintResultBo(Nothing)
+        '''        PrintResultSB(Nothing)
+        '''        PrintResultBy(Nothing)
+        '''        PrintResultSh(Nothing)
+        '''        PrintResultUs(Nothing)
+        '''        PrintResultIn(Nothing)
+        '''        PrintResultUI(Nothing)
+        '''        PrintResultLo(Nothing)
+        '''        PrintResultUL(Nothing)
+        '''        Prin [rest of string was truncated]&quot;;.
+        '''</summary>
+        Friend Shared ReadOnly Property ConversionsILGenTestSource2() As String
+            Get
+                Return ResourceManager.GetString("ConversionsILGenTestSource2", resourceCulture)
+            End Get
+        End Property
+        
+        '''<summary>
+        '''  Looks up a localized string similar to &lt;Global&gt;
+        '''            &lt;type name=&quot;&amp;lt;Module&amp;gt;&quot; /&gt;
+        '''            &lt;type name=&quot;A&quot; Of=&quot;T&quot; base=&quot;System.Object&quot;&gt;
+        '''              &lt;field name=&quot;x1&quot; type=&quot;A(Of T)&quot; /&gt;
+        '''              &lt;field name=&quot;x2&quot; type=&quot;A(Of D)&quot; /&gt;
+        '''              &lt;type name=&quot;B&quot; base=&quot;A(Of T)&quot;&gt;
+        '''                &lt;field name=&quot;y1&quot; type=&quot;A(Of T).B&quot; /&gt;
+        '''                &lt;field name=&quot;y2&quot; type=&quot;A(Of D).B&quot; /&gt;
+        '''                &lt;type name=&quot;C&quot; base=&quot;A(Of T).B&quot; /&gt;
+        '''              &lt;/type&gt;
+        '''              &lt;type name=&quot;H&quot; Of=&quot;S&quot; base=&quot;System.Object&quot;&gt;
+        '''              [rest of string was truncated]&quot;;.
+        '''</summary>
+        Friend Shared ReadOnly Property EmitSimpleBaseLine1() As String
+            Get
+                Return ResourceManager.GetString("EmitSimpleBaseLine1", resourceCulture)
+            End Get
+        End Property
+    End Class
+End Namespace
```

Comment:
Did anything actually change here, or did the line endings or encoding accidentally get changed?